src/device-registry/routes/v2/kya.js

```diff
@@ -1902,10 +1907,20 @@ router.put(
         .bail()
         .trim(),
       body("content")
-        .optional()
-        .notEmpty()
-        .withMessage("the content should not be empty")
-        .trim(),
+        .isArray()
+        .withMessage("content should be an array")
+        .bail()
+        .custom((value) => {
+          if (!Array.isArray(value)) {
```

Comment:
@BenjaminSsempala ...the repetition is still present but we shall check it out in a future PR I guess

Reply:
That's odd.
Sure thing I'll remember to remove it in a future pr.
@Baalmart

Reply:
Ooooh I didn't remove it in the validation for update,
Only the one for create, I'll handle this in a different PR. Thanks @Baalmart